lib/tpm2_policy.h

```diff
@@ -58,6 +58,29 @@ bool tpm2_policy_build_pcr(TSS2_SYS_CONTEXT *sapi_context,
         const char *raw_pcrs_file,
         TPML_PCR_SELECTION *pcr_selections);
 
+/**
+ * Enables a signing authority to authorize policies 
+ * @param sapi_context 
+ *   The system api context
+ * @param policy_session 
+ *   The policy session that has the policy digest to be authorized
+ * @param to_authorize_policy_file 
+ *   The policy digest file that needs to be authorized by signing authority
+ * @param to_authorize_policy_qualifier_file 
+ *   The policy qualifier data that concatenates with approved policies
+ * @param verifying_public_key_name_file 
+ *   The name of the public key that verifies the signature of the signer
+ * @param[in] verification_ticket_file 
```

Comment:
Is this valid doxygen?

Yes it is, wow: https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdparam

We should be using this everywhere from now on.